modules/src/crypto.py

```diff
@@ -0,0 +1,41 @@
+import requests
+
+from templates.text import TextTemplate
+
+
+def process(input, entities):
```

Comment:
It would be nice if we do not use inbuilt keywords and function as a variable :)

Reply:
@horrormyth would `entry` be a good replacement for `input`? (:

Reply:
user_input/entry would be fine, it is always better to have the relevant variable name so that it is very readable so to track the code. :)